napari/_qt/dialogs/qt_reader_dialog.py

```diff
@@ -67,9 +67,18 @@ def setup_ui(self, error_message, readers):
 
         # checkbox to remember the choice (doesn't pop up for folders with no extension)
         if self._extension:
-            self.persist_checkbox = QCheckBox(
-                f'Remember this choice for files with a {self._extension} extension'
-            )
+            if (
+                '*' + self._extension
+                in get_settings().plugins.extension2reader.keys()
+            ):
+                pref = get_settings().plugins.extension2reader[
+                    '*' + self._extension
+                ]
+                warn_message = f'Override existing preference for *{self._extension}, "{pref}"'
+            else:
+                warn_message = f'Remember this choice for files with a {self._extension} extension'
```

Comment:
Please add a translation of these strings.

Reply:
thanks for catching that, @Czaki !